homework-submission/react-hw4/src/components/stores.js

```diff
@@ -0,0 +1,39 @@
+import { observable, action, computed, configure } from "mobx";
+import Todos from "./todos.json";
+
+configure({ enforceActions: true });
+
+class Stores {
+  @observable
+  todos = Todos;
+
+  @action
+  checker = item => {
```

Comment:
This action doesn't update any observable in the store!?, so why did you create it?